lib/modules/manager/bun/index.ts

```diff
@@ -13,7 +13,7 @@ export const supersedesManagers = ['npm'];
 export const supportsLockFileMaintenance = true;
 
 export const defaultConfig = {
-  fileMatch: ['(^|/)bun\\.lockb?$'],
+  fileMatch: ['(^|/)bun\\.lockb?$', '(^|/)package\\.json$'],
```

Comment:
@rarkins  Do you remember why the lockfile is here in the filematch and not the package file?

Reply:
@secustor my goal originally had been to trigger bun logic only if there's a bun lockfile, and not for the 99% of cases where there's non-bun package.json files. However, I think this PR is the most pragmatic way of supporting workspaces in bun for now.

The alternative to changing `fileMatch` I considered is:
- Passing the include/ignore files list through to the extractAllPackageFiles function
- Letting the extractAllPackageFiles function search for related files within that filtered list
- etc

(If we didn't pass the result if include/ignore per-manager then we can end up with extracted results which don't honor the config)

But that seemed quite complicated too.